app/models/rectangles.model.js

```diff
@@ -11,15 +11,38 @@ var Rectangle = Backbone.Model.extend({
     initialize: function () {
         console.log('rectangle has been initialized');
 
-        // on attribute change calculates diagonal coordinates of rectangle
-        this.on('change', this.calcCoord)
+        // on attribute change parse to number and calculates diagonal coordinates of rectangle
+        this.on('change', this.parseToNumber)

```

Comment:
method name should cover logic inside. for this case I will wait that it is only parse number. no `.set()`